### PR TITLE
Swap sequitur g2p for fairseq g2p

### PR DESCRIPTION
**Pull request by judyfong**
We change out the IPA and aIPA phonemes for the x-SAMPA phonemes used in the latest version of the icelandic g2p models.

I keep getting this error: 
`./festival/feats/E.feats: bad value E in field p.name vector 3993`

Also, I couldn't upgrade my pip package on the xenial docker image so I swapped that image out for the python 3.7-stretch image which seems to work fine.

After deleting the repo and starting from scratch this branch works again. I still get the p.name vector error but it is now able to output a working utterance.

### CODE REVIEW

**Comment by judyfong** [comment]
Was there another file/place I should have modified?

**Comment by judyfong** [comment]
If possible, I'd like to git tag the current version in master as ipa and tag this version as sampa. What do you say? Also, this PR is ready to be reviewed now.

**Comment by thdg** [comment]
I created a release for milestone 4 tagged like was suggested by Anna.
We can edit the description of that for more specific comments like what g2p alphabet is used.

**Comment by judyfong** [comment]
OK. Yes editing the release description works.

File: voice/fairseq_g2p.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        # Process phrase to work with g2p functioon
+        # TODO: remove punctuation because it affects the output
+        # phrase = 'Velkomin til íslands.'
+        # phrase = 'Velkomin til íslands'
+        phrase = 'What is up Charlie Zinger Queen'
+        # Change a phrase to a list of words with .split()
+        phrase_spaced = words2spaced(phrase.split())
+
+        # Process words to work with g2p function
+        h_l_a_u_p_a = words2spaced(['hlaupa'])
```
Comment (thdg) [inline voice/fairseq_g2p.py:64]: Though it is fairly descriptive, that variable name is horrible! :rofl:

**Comment by thdg** [comment]
Looks good and runs fine.
The voice I got did not sound good so some tweaking might be necessary.

**Pull request merged**